pkg/client/client.go

```diff
@@ -216,6 +216,12 @@ func (c *Client) Run(ctx context.Context) {
 	}
 
 	wg.Wait()
+	if !c.p.GetBool(prop.DoTransactions, true) {
+		// when loading is finished, try to analyze table if possible.
+		if analyzeDB, ok := c.db.(ycsb.AnalyzeDB); ok {
```

Comment:
If I remember correctly, db here is DBWrapper, so we can call Analyze directly, maybe.

Reply:
`c.db` is of the `ycsb.DB` interface, so type assertion is required for `DBWrapper`:

https://github.com/pingcap/go-ycsb/blob/65aa46406f05ab9126991eec855b476c608985d6/pkg/client/client.go#L158